sql/server/src/main/java/io/stargate/api/sql/schema/StargateSchema.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.api.sql.schema;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.apache.calcite.linq4j.tree.Expression;
+import org.apache.calcite.rel.type.RelProtoDataType;
+import org.apache.calcite.schema.*;
+import org.immutables.value.Value;
+
+@Value.Immutable(prehash = true)
+public abstract class StargateSchema implements Schema {
```

Comment:
I think the naming is confusing, because stargate already has a `Schema` (in `persistence-api`). As this exists (afaict) only because Calcite wants _its_ version of the schema, I'd call it `CalciteSchema`, but `SQLSchema` probably works as well (avoids any confusion).

Reply:
Calcite already has a `CalciteSchema` in its JDBC packages :thinking: I'll go with `SQLSchema`

Reply:
renamed to `SqlSchema`